### PR TITLE
Issue 661 transaction callback

### PR DESCRIPTION
**Pull request by iammatis**

### CODE REVIEW

File: src/Model/Asset/AssetOptions.php [hunk 1]
```diff
@@ -92,12 +93,10 @@ public function toArray(): array
     public function toBytes(): string
     {
         return implode('', [
-            str_pad(Math::gmpDecHex(Math::reverseBytesLong($this->getMaxSupply())), 16, '0', STR_PAD_LEFT),
+            Math::getInt64($this->getMaxSupply()),
             $this->getExchangeRate()->toBytes(),
             $this->isExchangeable() ? '01' : '00',
-            str_pad(count($this->getExtensions()), 2, '0', STR_PAD_LEFT),
-            // TODO: isFixedMaxSupply
-//            '01'
```
Comment (ttibensky) [inline src/Model/Asset/AssetOptions.php]: unresolved todo

File: src/Model/Asset/MonitoredAssetOptions.php [hunk 2]
```diff
@@ -138,12 +139,14 @@ public function toArray(): array
     public function toBytes(): string
     {
         return implode('', [
-            // TODO: Feeds Array
-            $this->getFeeds() ? '01' : '00',
+            $this->getFeeds() ?
+                VarInt::encodeDecToHex(sizeof($this->getFeeds()))
+                . '' // TODO array_map each element toBytes()
```
Comment (ttibensky) [inline src/Model/Asset/MonitoredAssetOptions.php]: unresolved todo

File: src/Model/Operation/AssetClaimFeesOperation.php [hunk 3]
```diff
@@ -113,7 +114,10 @@ public function toBytes(): string
             $this->getUia()->toBytes(),
             $this->getDct()->toBytes(),
             // TODO: Extensions Array
-            $this->getExtensions() ? '01' : '00'
+            $this->getExtensions() ?
+                VarInt::encodeDecToHex(sizeof($this->getExtensions()))
+                . '' // TODO array_map for every element in array
```
Comment (ttibensky) [inline src/Model/Operation/AssetClaimFeesOperation.php]: unresolved todo

File: src/Model/Operation/AssetCreateOperation.php [hunk 4]
```diff
@@ -187,16 +188,18 @@ public function toBytes(): string
...
+            VarInt::encodeDecToHex(sizeof(Math::stringToByteArray($this->getDescription()))),
             Math::byteArrayToHex(Math::stringToByteArray($this->getDescription())),
             $this->getOptions()->toBytes(),
             $this->getMonitoredOptions() ? $this->getMonitoredOptions()->toBytes() : '00',
             '01',
-            // TODO: Extensions Array
-            $this->getExtensions() ? '01' : '00'
+            $this->getExtensions() ?
+                VarInt::encodeDecToHex(sizeof($this->getExtensions()))
+                . '' // TODO array_map each element toBytes()
```
Comment (ttibensky) [inline src/Model/Operation/AssetCreateOperation.php]: unresolved todo

File: src/Model/Operation/AssetFundPoolsOperation.php [hunk 5]
```diff
@@ -111,8 +112,10 @@ public function toBytes(): string
             $this->getFrom()->toBytes(),
             $this->getUia()->toBytes(),
             $this->getDct()->toBytes(),
-            // TODO: Extensions Array
-            $this->getExtensions() ? '01' : '00'
+            $this->getExtensions() ?
+                VarInt::encodeDecToHex(sizeof($this->getExtensions()))
+                . '' // TODO array_map each element toBytes()
```
Comment (ttibensky) [inline src/Model/Operation/AssetFundPoolsOperation.php]: unresolved todo

File: src/Model/Asset/AssetOptions.php (hunk 1 above)
Comment (iammatis) [reply src/Model/Asset/AssetOptions.php]: This is a deleted code, aka has been fixed and this section doesn't exist anymore.

File: src/Model/Asset/MonitoredAssetOptions.php (hunk 2 above)
Comment (iammatis) [reply src/Model/Asset/MonitoredAssetOptions.php]: Unknown structure of array, can't fix now.

File: src/Model/Operation/AssetCreateOperation.php (hunk 4 above)
Comment (iammatis) [reply src/Model/Operation/AssetCreateOperation.php]: Unknown structure of array, can't fix now.

File: src/Model/Operation/AssetClaimFeesOperation.php (hunk 3 above)
Comment (iammatis) [reply src/Model/Operation/AssetClaimFeesOperation.php]: Unknown structure of array, can't fix now.

File: src/Model/Operation/AssetFundPoolsOperation.php (hunk 5 above)
Comment (iammatis) [reply src/Model/Operation/AssetFundPoolsOperation.php]: Unknown structure of array, can't fix now.

**Pull request merged**